fedbiomed/researcher/federated_workflows/jobs/_job.py

```diff
@@ -27,46 +25,70 @@ class Job(ABC):
         the execution of the task.
 
     Attributes:
-        requests: read-only [`Requests`][fedbiomed.researcher.requests.Requests] object handling communication with remote nodes
+        requests: read-only [`Requests`][fedbiomed.researcher.requests.Requests] object handling communication with
+                 remote nodes
         nodes: node IDs participating in the task
     """
 
     def __init__(self,
-                 *,
```

Comment:
Thank you @sharkovsky 
Yes, I confirm this is intentional to have only named arguments to avoid confusion + avoid ordering constraints for named/positional arguments.
Discussing with @ybouilla : his modification is unwanted, this is a side effect of back merging in this branch